src/main/java/ch/hearc/cafheg/business/allocations/AllocationService.java

```diff
@@ -154,15 +164,17 @@ public void deleteAllocataire(long allocataireId) {
      * @throws RuntimeException             if the update fails in the database for other reasons.
      */
     public Allocataire updateAllocataire(long allocataireId, String newNom, String newPrenom) {
-        System.out.println("Service: Tentative de mise à jour du nom de l'allocataire avec ID: " + allocataireId);
+        logger.info("Service: Tentative de mise à jour du nom/prénom de l'allocataire avec ID: {}", allocataireId);
 
         // Validate input
         if (newNom == null || newNom.trim().isEmpty() || newPrenom == null || newPrenom.trim().isEmpty()) {
+            logger.warn("Le nom ou le prénom fourni est vide pour l'allocataire ID: {}", allocataireId);
             throw new IllegalArgumentException("Le nom et le prénom de l'allocataire ne peuvent pas être vides.");
         }
 
         Allocataire existingAllocataire = allocataireMapper.findById(allocataireId);
         if (existingAllocataire == null) {
+            logger.error("Allocataire non trouvé avec ID: {}", allocataireId);
```

Comment:
Same here, not a critical error
```suggestion
            logger.warn("Allocataire non trouvé avec ID: {}", allocataireId);
```